lib/teecket/air_asia.rb

```diff
@@ -21,15 +26,15 @@ def get
     req.add_field("Channel", "mobile-touch-app")
 
     req.body = URI.encode_www_form([
-      ["type", "classic"],
+      %w(type classic),
```

Comment:
How about keeping it as array element as others are using array, for consistency